crates/generator/src/cli.rs

```diff
@@ -27,23 +27,23 @@ impl CliConfig {
                     .short('d')
                     .value_name("DEGREE")
                     .help("Cyclotomic polynomial degree (power of 2)")
-                    .default_value("1024"),
+                    .default_value("2048"),
             )
             .arg(
                 Arg::new("plaintext-modulus")
                     .long("plaintext-modulus")
                     .short('t')
                     .value_name("MODULUS")
                     .help("Plaintext modulus")
-                    .default_value("2048"),
+                    .default_value("1032193"),
```

Comment:
Hey @0xjei, do you recall where you got this number? I think the last digit should be 2, which might be the source of the problem. Could you confirm if this is a typo?